examples/basic/tidb-monitor.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   clusters:
   - name: basic
+    namespace: default
```

Comment:
after this (https://github.com/pingcap/tidb-operator/pull/1834), the namespace can be omitted now.

the benefit is users can test our example in any namespace they like, `kubectl apply -n <ns>  examples/basic/`.